packages/app/components/notifications/index.tsx

```diff
@@ -121,10 +121,6 @@ export const Notifications = ({
             ios: windowHeight,
           }),
         }}
-        overscan={{
```

Comment:
hardcoding overscan to 2 in the list component itself. It will render 2 additional items above and below

Reply:
I think it should still be overwritable on a component basis, because specially on short lists it acts a bit weird when it messes with the elements. It breaks momentum scrolling and stops scrolling.